psychopy/data/experiment.py

```diff
@@ -335,6 +334,18 @@ def saveAsPickle(self, fileName, fileCollisionMethod='rename'):
         logging.info('saved data to %s' % f.name)
         self.savePickle = savePickle
         self.saveWideText = saveWideText
+        
+    def close(self):
+        if self.dataFileName not in ['', None]:
+            if self.autoLog:
+                logging.debug(
+                              'Saving data for %s ExperimentHandler' % self.name)
+            if self.savePickle == True:
+                self.saveAsPickle(self.dataFileName)
+            if self.saveWideText == True:
+                self.saveAsWideText(self.dataFileName + '.csv', delim=',')
+        self.abort()
```

Comment:
Do we really still need to call `abort()`?

Reply:
I put the abort call because we kept the '__del__' method. This way if the close method is called explicitly then the ExperimentHandler won't try to rewrite the output files if that instance is later deleted (since deleting it now calls '.close()')